### PR TITLE
GRVDEV- 86 Add S3 Import Support to RDS Postgres Module

### PR DESCRIPTION
**Pull request by TysonBeck**
-Added S3 Import IAM and attachment to RDS instance
-Added support for storage autoscaling
-Added support for performance insights
-Removed remote state data

### CODE REVIEW

File: aws/rds-postgres/db.tf [hunk 1]
```diff
@@ -321,21 +319,24 @@ data "aws_kms_key" "parameter_store_key" {
...
   kms_arn        = "alias/parameter_store_key"
-  parameter_read = ["/${local.module_prefix}/rds-secret"]
+  parameter_read = ["/${local.stage_prefix}/${var.name}-password"]
 }
 
 module "rds_ssm_param_username" {
+  enabled        = var.username == "" ? "true" : "false"
   source         = "git::https://github.com/cloudposse/terraform-aws-ssm-parameter-store?ref=0.1.5"
-  parameter_read = ["/${local.module_prefix}/rds-username"]
+  parameter_read = ["/${local.stage_prefix}/${var.name}-username"]
```
Comment (jeremyodle) [inline aws/rds-postgres/db.tf:324]: local.module_prefix should work here

File: aws/rds-postgres/db.tf [hunk 2]
```diff
@@ -1,13 +1,13 @@
 # ----------------------------------------------------------------------------------------------------------------------
 # VARIABLES / LOCALS / REMOTE STATE
 # ----------------------------------------------------------------------------------------------------------------------
-variable "terraform_remote_state_vpc_key" {
-  description = "Key for the location of the remote state of the vpc module"
-}
+# variable "terraform_remote_state_vpc_key" {
```
Comment (jeremyodle) [inline aws/rds-postgres/db.tf]: Remove commented code if no longer needed

**Review by jeremyodle — CHANGES_REQUESTED** [review]

File: aws/rds-postgres/db.tf (hunk 1 above)
Comment (TysonBeck) [reply aws/rds-postgres/db.tf:324]: Unable to use module prefix due to the need to add the separating "/" as name is not part of the parameter service.

File: aws/rds-postgres/db.tf (hunk 2 above)
Comment (TysonBeck) [reply aws/rds-postgres/db.tf]: Removed commented code

**Review by jeremyodle — APPROVED** [review]

**Pull request merged**